xml/schema.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE schema SYSTEM "../../../../lib/pkp/dtd/xmlSchema.dtd">
+
+<!--
+plugins/generic/betterPassword/xml/schema.xml
+
+betterPassword plugin schema in XML
+-->
+
+
+<schema version="0.2">
```

Comment:
In creating this new schema, we should migrate the existing entries in the user record to this new datastructure.  Unfortunately, to actually do so in a plugin is way more complicated than it needs to be, so we can defer this for now unless you want a challenge.